pkg/apis/migration/v1alpha1/migstorage_types.go

```diff
@@ -99,7 +99,7 @@ func (r *MigStorage) Equals(a, b *velero.BackupStorageLocation) bool {
 func (r *MigStorage) BuildBSL() *velero.BackupStorageLocation {
 	location := &velero.BackupStorageLocation{
 		ObjectMeta: metav1.ObjectMeta{
-			Labels:       CorrelationLabels(r, r.Namespace, r.Name),
+			Labels:       CorrelationLabels(r, r.UID),
```

Comment:
Am I correct in understanding that this will change the format of the CorrelationLabels from:

`"owner_kind": "owner_ns/owner_name"`
to...
`"owner_kind": "owner_uid"`

Reply:
Answered my own question by looking a little further in the PR.